### PR TITLE
Fix/field bug

### PR DESCRIPTION
**Pull request by nDny**
Fix breaking changes in django 4.2.1

Also add helper classes to not make people suicidal when only being able to upload 1 image at a time.

### CODE REVIEW

File: archive/forms.py [hunk 1]
```diff
@@ -3,24 +3,40 @@
...
+            result = [single_file_clean(d, initial) for d in data]
+        else:
+            result = single_file_clean(data, initial)
+        return result
+
+
 class PictureUploadForm(forms.Form):
     album = forms.CharField()
-    images = forms.ImageField(required=False, widget=forms.ClearableFileInput(attrs={'multiple': True}))
+    images = forms.ImageField(required=False, widget=forms.ClearableFileInput())
```
Comment (andreedd) [inline archive/forms.py]: Should it use the MultipleFileInput?

Comment (nDny) [reply archive/forms.py]: Sure, but does anyone even use that one lol :D Is that not the one obfuscated somewhere in the archive view

Comment (nDny) [reply archive/forms.py]: Yup this one 
image

Comment (jaspnas) [reply archive/forms.py]: > Sure, but does anyone even use that one lol :D Is that not the one obfuscated somewhere in the archive view

I think I saw someone use that one at some point

File: archive/forms.py [hunk 2]
```diff
@@ -3,24 +3,40 @@
...
 
 
 class ExamUploadForm(forms.Form):
     title = forms.CharField()
-    exam = forms.FileField(required=False, widget=forms.ClearableFileInput(attrs={'multiple': False}))
+    exam = forms.FileField(required=False, widget=forms.ClearableFileInput())
```
Comment (andreedd) [inline archive/forms.py]: Was thinking for these to use multiple as well? `MultipleFileInput`

File: archive/forms.py [hunk 3]
```diff
@@ -36,7 +52,7 @@ def save(self, *args, **kwargs):
 
 
 class DocumentAdminForm(forms.ModelForm):
-    files = forms.FileField(widget=forms.ClearableFileInput(attrs={'multiple': True}),
+    files = forms.FileField(widget=forms.ClearableFileInput(),
```
Comment (andreedd) [inline archive/forms.py]: Was thinking for these to use multiple as well? `MultipleFileInput`

File: archive/forms.py [hunk 4]
```diff
@@ -54,7 +70,7 @@ def save(self, *args, **kwargs):
         return collection
 
 class PublicAdminForm(forms.ModelForm):
-    files = forms.FileField(widget=forms.ClearableFileInput(attrs={'multiple': True}),
+    files = forms.FileField(widget=forms.ClearableFileInput(),
```
Comment (andreedd) [inline archive/forms.py]: Was thinking for these to use multiple as well? `MultipleFileInput`

Comment (andreedd) [reply archive/forms.py]: This is for uploading public s3 files, used when making news posts with public images for example

File: archive/forms.py (hunk 2 above)
Comment (andreedd) [reply archive/forms.py]: This so Hektor does not have to upload single :D

**Comment by nDny** [comment]
Yeh probably makes sense, I only put it on the most critical one because Django docs said that it **_should_** be a temporary fix and that they will implement a proper multi choice input later.

**Review by andreedd — APPROVED** [review]

**Pull request merged**